artman/cli/main.py

```diff
@@ -46,6 +48,13 @@
 ARTMAN_DOCKER_IMAGE = 'googleapis/artman:%s' % VERSION
 RUNNING_IN_ARTMAN_DOCKER_TOKEN = 'RUNNING_IN_ARTMAN_DOCKER'
 
+# Error messages
+CONFIG_NOT_FOUND_ERROR_MESSAGE_FORMAT = (
+    'Artman YAML cannot be found at `%s`. Please check the file location.')
+INVALID_CONFIG_ERROR_MESSAGE_FORMAT = (
+    'Artman YAML %s is invalid. Please pass a valid Artman YAML (e.g. '
```

Comment:
I would rather print the cause of the parsing error (line number etc.), assuming the yaml library can provide this. I'd prefer not to link to the master branch of a GitHub repository in the error message, which is brittle if we refactor googleapis in the future, etc.

Reply:
Removed. Added a TODO to point to a formal artman config page link once we have one.